cli/src/main/kotlin/utils/SeverityStats.kt

```diff
@@ -77,17 +77,17 @@ internal class SeverityStats(
         val resolvedErrorCount = getResolvedCount(Severity.ERROR)
 
         println(
-            "Found $resolvedErrorCount resolved error(s), $resolvedWarningCount resolved warning(s), " +
-                    "$resolvedHintCount resolved hint(s)."
+            "Resolved issues: $resolvedErrorCount error(s), $resolvedWarningCount warning(s), " +
```

Comment:
I was avoiding the word "issue" here because this is also used by the evaluator where it relates to rule violations, not issues. I think we could have to give the `SeverityStats` some property to set the source of the severerities to be able to print the correct message here.

Reply:
Good point. That reminds me of a [discussion](https://github.com/oss-review-toolkit/ort/pull/4454#discussion_r707358332) we had (and it's also an item on the dev meeting's [agenda](https://github.com/oss-review-toolkit/ort/wiki/Developer-Meeting#agenda)) whether the `severeIssueThreshold` should even apply to violations, as violations are not issues (which is something I tend to forget myself), and while the severity of issues in hard-coded in ORT, the severity of violations is user-defined by the rules.

So, how about addressing this by making `severeIssueThreshold` *not* to be used when the evaluator prints its stats?

Reply:
I think it would be better to have a separate `severeRuleViolationThreshold`, but let's wait for the outcome of the dicussion in the dev meeting.

Reply:
Let's continue the discussion at https://github.com/oss-review-toolkit/ort/issues/4642.